cpt-repo.tex

```diff
@@ -455,6 +455,32 @@ \subsection{Managing Nested Catalogs with \texttt{.cvmfsdirtab}}
 \end{verbatim}
 Note that unlike the regular lines that add catalogs, asterisks in the exclamation point exclusion lines can span the slashes separating directory levels.
 
+\subsection{Automatic Management of Nested Catalogs}
+\label{sct:autocatalogs}
+An alternative to \texttt{.cvmfsdirtab} is the automatic catalog generation.
+This feature automatically generates nested catalogs based on their weight (number of entries).
+It can be enabled by setting \texttt{CVMFS\_AUTOCATALOGS=true} in the server configuration file.
+
+Catalogs are split when their weight is greater than a specified maximum threshold, or removed if their weight is less than a minimum threshold.
+Autogenerated catalogs contain a \texttt{.cvmfsautocatalog} file (along with the \texttt{.cvmfscatalog} file) in its root directory.
+User-defined catalogs (containing only a \texttt{.cvmfscatalog} file) always remain untouched.
+Hence you can intermix both manual and automatically managed directory sub-trees.
+
+The following conditions are applied when processing a nested catalog:
+\begin{itemize}
+  \setlength{\itemsep}{1pt}
+  \item If the weight is greater than \texttt{CVMFS\_AUTOCATALOGS\_MAX\_WEIGHT}, this catalog will be split in smaller catalogs that meet the maximum and minimum thresholds.
+  \item If the weight is less than \texttt{CVMFS\_AUTOCATALOGS\_MIN\_WEIGHT}, this catalog will be merged into its parent.
+\end{itemize}
+For instance, a possible configuration might be:
+\begin{verbatim}
+CVMFS_AUTOCATALOGS=true
+CVMFS_AUTOCATALOGS_MAX_WEIGHT=100000
+CVMFS_AUTOCATALOGS_MIN_WEIGHT=1000
+\end{verbatim}
+
+Note that both \texttt{CVMFS\_AUTOCATALOGS\_MAX\_WEIGHT} and \texttt{CVMFS\_AUTOCATALOGS\_MIN\_WEIGHT} have reasonable defaults and do not need to be defined by the user.
```

Comment:
I'd like to see the defaults explicitly listed in the documentation.  

Also, I'll note that this section will also need to go into http://cernvm.cern.ch/portal/filesystem/maintain-repositories after it is released.